dapps/admin/src/utils/discovery.js

```diff
@@ -0,0 +1,104 @@
+import { getDiscovery } from './config'
+
+/**
+ * Utilities for querying, and authenticating with, the discovery server.
+ */
+
+// Used for localStorage
+const TOKEN_STORAGE_KEY = 'discoveryAuthToken'
+
+/**
+ * Send a query to the discovery server.
+ * @param {*} query
+ * @param {*} variables
+ * @returns {Promise<*>} Results of query
+ */
+export function query(query, variables) {
+  const authToken = activeAuthToken()
+  return new Promise(function(resolve, reject) {
+    const headers = {
+      'Content-Type': 'application/json',
+      Accept: 'application/json'
+    }
+    if (authToken) {
+      headers['x-discovery-auth-token'] = authToken
+    }
+    fetch(getDiscovery(), {
```

Comment:
Ideally we'd have the discovery GraphQL server merged in the same way as Growth is, but this works for now 😊